src/main/java/com/wseemann/ecp/core/HttpManager.kt

```diff
@@ -0,0 +1,57 @@
+package com.wseemann.ecp.core
+
+import java.util.concurrent.TimeUnit
+
+import okhttp3.OkHttpClient
+import okhttp3.ConnectionPool
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.ResponseBody
+import okhttp3.Request
+import okhttp3.RequestBody.Companion.toRequestBody
+
+class HttpManager private constructor() {
+
+    private val okHttpClient = OkHttpClient.Builder()
```

Comment:
These are extremely low connection and read times. I'd prefer at least two seconds. While these times may work on your internet connection other users may have less than ideal network conditions are setting the values this low would make this SDK unusable.